docs/admin-ports.md

```diff
@@ -0,0 +1,30 @@
+# Admin Ports
+
+The following is a list of TCP ports used by internal DC/OS services, and their corresponding systemd unit.
+
+## All Roles
+
+ - 1050: dcos-ddt
```

Comment:
I'll just rename the unit to make it clearer: https://github.com/dcos/dcos/pull/233